openassessment/templates/openassessmentblock/response/oa_response_graded.html

```diff
@@ -26,9 +25,7 @@
     <div class="wrapper--step__content">
         <div class="step__content">
             <article class="submission__answer__display">
-                <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
-
-                {% trans "Your response to the question above:" as translated_label %}
+                {% trans "Your response" as translated_label %}
```

Comment:
When is this template used? I'm wondering why "Your response" is not in a header here, as it usually is.